src-ui/lib/components/BooksTable.svelte

```diff
@@ -79,19 +82,21 @@
     <thead>
       <tr>
         <th><!-- delete action --></th>
-        <th>ISBN</th>
+        <th>isbn10</th>
+        <th>isbn13</th>
         <th>Title</th>
         <th>Author</th>
         <th>Tags</th>
         <th>Read?</th>
+        <th>Scan Date</th>
       </tr>
     </thead>
     <tbody>
       {#each booksStore.value as book}
         <BooksTableRow {book} />
       {:else}
         <tr>
-          <td colspan="5">
+          <td colspan="6">
```

Comment:
Should this be `8`?
```suggestion
          <td colspan="8">
```

Reply:
Makes sense since there are 8 columns; but why was it 5 before?  Perhaps an oversight?

Reply:
yeah, was just wrong :)